source/jormungandr/jormungandr/interfaces/v1/serializer/pt.py

```diff
@@ -309,13 +314,16 @@ class CarParkSerializer(PbNestedSerializer):
     total_places = jsonschema.IntField()
 
 
-class AdminSerializer(PbGenericSerializer):
+class AdminSerializer(SortedGenericSerializer, PbGenericSerializer):
```

Comment:
Don't you end up with some Diamond inheritance problem ? 
Both SortedGenericSerializer and PbGenericSerializer inherits from serpy.Serializer

Reply:
let me see

Reply:
https://www.programiz.com/python-programming/multiple-inheritance

take a look at "MRO" :)